Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -740,16 +740,10 @@ class MainMenuActionHelper: PhotonActionSheetProtocol, FeatureFlaggable, CanRemo
                                      iconString: ImageIdentifiers.addShortcut) { _ in
 
             guard let url = self.selectedTab?.url?.displayURL,
-                  let sql = self.profile.history as? SQLiteHistory
+                  let title = self.selectedTab?.title
```

Comment:
Hmm can the title be nil? If so then we can use display title as that won't be nil afaik.

Actually the short title is what we should be using so basically instead of full url we only take the website name from full url 

https://www.mozilla.com becomes `mozilla` for title

Let's double check if display title gives shirt url title so we don't end up displaying full url in title.

Reply:
Switched it to using the display title! there is logic in https://github.com/mozilla-mobile/firefox-ios/blob/aa9120e7d55729157e275558c332aa754c3881fb/Client/Frontend/Browser/Tab%20Management/Tab.swift#L178-L209 that seems to have a smart fallback mechanism!